modules/local/interproscan_download.nf

```diff
@@ -0,0 +1,35 @@
+process INTERPROSCAN_DATABASE {
+    tag "interproscan_database_download"
+    label 'process_medium'
```

Comment:
```suggestion
    label 'process_long'
```
Maybe?

Reply:
@Darcy220606 Do we need 6 cores for database extraction? If not, I agree with the suggestion to reduce cores and memory:

```
withLabel: process_medium {
    cpus   = { 6 * task.attempt }
    memory = { 36.GB * task.attempt }
    time   = { 8.h * task.attempt }
}
withLabel: process_long {
    time = { 20.h * task.attempt }
}
```

Reply:
Decided to use `process_long` but leave custom cpu/memory configuration in base.config.